viewer/packages/camera-manager/src/Flexible/FlexibleCameraMarkers.ts

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import { Mesh, MeshBasicMaterial, SphereGeometry, Scene, Object3D, Vector3 } from 'three';
+import { ControlsType } from './ControlsType';
+import { FlexibleCameraManager } from './FlexibleCameraManager';
+
+export class FlexibleCameraMarkers {
+  private readonly _scene?: undefined | Scene;
+  private _pivotMarker: Object3D | undefined;
+  private _lookAtMarker: Object3D | undefined;
+
+  //================================================
+  // CONSTRUCTOR
+  //================================================
+
+  constructor(scene?: Scene) {
```

Comment:
Does `scene` need to be optional? Can we instead just not construct the markers if no scene is available?